opus/application/static_media/js/hash.js

```diff
@@ -641,6 +641,13 @@ var o_hash = {
                     let slugCounter = o_utils.getSlugOrDataTrailingCounterStr(slug);
                     slug = slugNoCounter;
 
+                    // For surfacegeometrytargetname, we have to assign the init value
```

Comment:
I found this comment somewhat confusing, because I didn't remember that this function was only called on page reload.

How about "If there is a value specified for surfacegeometrytargetname in the URL, we have to initialize opus.oldSurfacegeoTarget with it so that we can properly handle changes to surfacegeometrytargetname in the future."